ext/NNlibCUDA/.buildkite/pipeline.yml

```diff
@@ -0,0 +1,20 @@
+steps:
+  - label: "GPU integration with julia v1.6"
+    plugins:
+      - JuliaCI/julia#v1:
+          version: "1.6" 
+      - JuliaCI/julia-test#v1: ~
+    agents:
+      queue: "juliagpu"
+      cuda: "*"
+    timeout_in_minutes: 60
+
+  - label: "GPU integration with julia v1"
+    plugins:
+      - JuliaCI/julia#v1:
+          version: "1.8"
+      - JuliaCI/julia-test#v1: ~
+    agents:
+      queue: "juliagpu"
+      cuda: "*"
+    timeout_in_minutes: 60
```

Comment:
not blocking for this PR, cleanup can be done later, but subpackages' `.buildkite` and `.github` dirs should be removed?

Reply:
Yes, absolutely.